python/hail/dataset.py

```diff
@@ -2940,6 +2940,84 @@ def query_variants(self, exprs):
         r, t = self.query_variants_typed(exprs)
         return r
 
+    def query_genotypes_typed(self, exprs):
+        """Perform aggregation queries over genotypes, and returns python objects and types.
+
+        **Examples**
+
+        >>> gq_hist, t = vds.query_genotypes_typed(
+        ...     'gs.map(g => g.gq).hist(0, 100, 100)')
+
+        See :py:meth:`.query_genotypes` for more information.
+
+        This method evaluates Hail expressions over genotypes, along with
+        all variant and sample metadata for that genotype. The ``exprs``
+        argument requires either a list of strings or a single string
+        (which will be interpreted as a list with one element).  The method
+        returns a list of results (which contains one element if the input
+        parameter was a single str).
+
+        The namespace of the expressions includes:
+
+        - ``global``: global annotations
+        - ``gs`` (*Aggregable[Genotype]*): aggregable of :ref:`genotype`
+
+        Map and filter expressions on this aggregable have the additional
```

Comment:
Additional seems wrong.  The aggregable namespace does not include the outer namespace.